project/Dependencies.scala

```diff
@@ -9,14 +9,15 @@ object Dependencies {
   val Scala212 = "2.12.17"
   val ScalaVersions = Seq(Scala213, Scala212)
 
-  val AkkaVersionInDocs = "2.7"
+  val AkkaVersionInDocs = Versions.akka
```

Comment:
wasn't the purpose to have links to 2.7 rather than the specific patch 2.7.0?

Reply:
Ah, was just bothered with having to change two, if we keep it 2.7.0 we will need to say "But for this feature you need 2.8" in the docs (which is maybe fine).